problem_builder/models.py

```diff
@@ -21,8 +21,16 @@
 # Imports ###########################################################
 
 from django.db import models
+from django.db.models.signals import pre_delete
 from django.contrib.auth.models import User
 
+try:
+    # workaround so we don't explicitly import the AnonymousUserId model from LMS
+    from student.models import AnonymousUserId
+    ANONYMOUS_USER_ID_IMPORTED = True
+except ImportError:
+    ANONYMOUS_USER_ID_IMPORTED = False
```

Comment:
Rather than have a separate variable for, you can do:

```python
try:
    # workaround so we don't explicitly import the AnonymousUserId model from LMS
    from student.models import AnonymousUserId
except ImportError:
    AnonymousUserId = None
...
if AnonymousUserId is not None:
    pre_delete.connect(delete_anonymous_user_answers, sender=AnonymousUserId)
```

Reply:
@xitij2000 Thanks! Yes, that does look better. I've updated the condition.